### ISSUE TITLE
Test version API by making Github releases

### ISSUE THREAD

**Issue by dmos62** [centerofci/mathesar#2485]
## Problem

Frontend queries Github Releases API (proxied through backend at the moment) for information about the current and latest releases. While that can be mocked (while we don't have Github releases), ultimately we want to test using the actual Github API, which means making Github releases. We'll naturally have a release when we make our first public release, but we probably want to test earlier than that, and we probably want two (for testing various scenarios).

Notably, having Github releases might facilitate frontend development, so it might make sense to do prereleases on Github as soon as possible. 

## Proposed solution

Create two mock releases on Github so that Github Releases API responds similarly to how it would respond when we have public/real releases.

## Additional context
- Blocked by #202

**Comment by dmos62** [comment]
@kgodey can you review for approval?

Tagging @seancolsen for visibility.

**Comment by kgodey** [comment]
Approved, but #202 should be done before doing the first release.

**Comment by dmos62** [comment]
@seancolsen assigning you, since you'll be testing this, as I understand.

**Comment by dmos62** [comment]
@seancolsen published two pre-releases on GitHub:

https://github.com/centerofci/mathesar/releases/tag/0.0.0
https://github.com/centerofci/mathesar/releases/tag/0.0.1

and two Docker images on dockerhub, both tagged `latest` and one additionally tagged `0.0.0` and the other `0.0.1`:

https://hub.docker.com/r/mathesar/mathesar-prod/tags

To be clear, nothing about these pre-releases is final. We can remove all of them when we want (if we want).

**Comment by dmos62** [comment]
Marking unblocked since https://github.com/centerofci/mathesar/issues/202 seems to be finished.

**Comment by kgodey** [comment]
@dmos62 I don't think you should close this issue until the upgrade is actually tested.

**Comment by kgodey** [comment]
Also, I don't understand how making two releases now is helpful for upgrade. I assume we need to make one release, deploy a server with that release, then make a second release, and test upgrade with that release.

**Comment by dmos62** [comment]
@kgodey you're right, I setup what's necessary for testing, but testing itself didn't happen. I could have made more granular issues. I'll reopen.

Another issue is that the images are broken (as notified by Brent early today), so we'll have to rebuild and repush once that's fixed.

> I don't understand how making two releases now is helpful for upgrade. I assume we need to make one release, deploy a server with that release, then make a second release, and test upgrade with that release.

You don't have to make the releases separately, afaik. You should be able to manipulate the versions of your images so that you're running the first version and use the in-app upgrade process to upgrade to the second one.

**Comment by kgodey** [comment]
Closing in favor of https://github.com/centerofci/mathesar/issues/2534
